raven/src/main/java/com/getsentry/raven/event/Breadcrumb.java

```diff
@@ -0,0 +1,111 @@
+package com.getsentry.raven.event;
+
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * An object that represents a single breadcrumb. Events may include a list
+ * of breadcrumbs that help users re-create the path of actions that occurred
+ * which lead to the Event happening.
+ */
+public class Breadcrumb {
+
+    /**
+     * (Optional) Type of the breadcrumb.
+     */
+    private final String type;
+    /**
+     * Timestamp when the breadcrumb occurred.
+     */
+    private final Date timestamp;
+    /**
+     * (Optional) Duration of the portion this breadcrumb represents.
+     */
+    private final float duration;
```

Comment:
I'm not really sure how to interpret this commit, but I think this may have been removed here: getsentry/sentry@fa7a4159ca778d7516f01a7a74e30d33ff0a8fa5